src/shared/layouts/BaseLayout/BaseLayout.tsx

```diff
@@ -18,22 +18,42 @@ export const BaseLayout = () => {
   const dispatch = useAppDispatch();
   const isAuthorized = auth.useAuthorized();
   const accountData = account.useData();
+  const currentWorkspaceData = workspaces.useData();
   const { duplicatePopupsVisible, deletePopupVisible, transferOwnershipPopupVisible } =
     popups.useData();
 
-  useEffect(() => {
-    if (isAuthorized) {
-      dispatch(users.thunk.getManagersList());
-      dispatch(users.thunk.getUsersList());
-    }
-  }, [dispatch, isAuthorized]);
-
   useEffect(() => {
     if (accountData?.account && isAuthorized) {
       dispatch(folders.thunk.getAppletsForFolders({ account: accountData.account }));
     }
   }, [dispatch, accountData?.account, isAuthorized]);
 
+  useEffect(() => {
+    const { getWorkspaceApplets } = applets.thunk;
+    const { getWorkspaceUsers } = users.thunk;
+
```

Comment:
const ownerId = currentWorkspaceData?.ownerId;
and pls use this variable to get rid of repeated code